code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -78,6 +78,12 @@
 	description = "Rare and experimental particles, that apparently swap the user's body with one from an alternate dimension where it's completely healthy."
 	taste_description = "science"
 
+/datum/reagent/medicine/adminordrazine/strangevodka
+	name = "Strange Vodka"
+	description = "An ancient mixture of extremely strong and rare reagents. Has the power to rapidly repair any and all wounds, at the cost of getting the user very, very drunk."
+	taste_description = "the motherland"
+	var/boozepwr = 20000
```

Comment:
No really this will not work. Boozepwr is from ethanol and anything descended from ethanol, but this is descended from adminorazine.

If you really want this to work, I suggest making this a descendant of ethanol that metabolizes into adminorazine. Take a look at how other reagents work to find out more.

Reply:
This won't even work as he expects because of how boozepwr interacts with drunkenness calculations, and afaik since adminordrazine works with any amount, you can keep diluting the mixture for infinite adminheals every sip.

Test your shit quite honestly